Expose more sensors

DavidWAbrahams opened the pull request:
Beyond just the power of each panel, report the grid frequency, grid voltage, and temperature. These are certainly less important than the voltage. But I do like having access to the temperature on my roof.

DavidWAbrahams committed:
Expose more sensors

Beyond just the power of each panel, report the grid frequency, grid voltage, and temperature. These are certainly less important than the voltage. But I do like having access to the temperature on my roof.

jeeshofone commented:
Found a small issue in the code, header of the table is ending up in the `inverter_ids` array so the first sensor in the config generator is using the value `Inverter ID` 

I'll commit a small change that fixes this and then we can merge.

jeeshofone committed:
Ignores the header in the row

Also update README.md to reflect the name of the populate-configuration.py file and add a .gitignore for the resulting configuration to keep it from ending up in the project.
